server/gorm/deletions.go

```diff
@@ -61,7 +61,7 @@ func (c *Client) DeleteChildrenOfProject(ctx context.Context, project *models.Pr
 		q = q.Require("ProjectID", project.ProjectID)
 		err := c.DeleteAllMatches(ctx, q)
 		if err != nil {
-			//return err
+			return err
```

Comment:
If the intention is to ignore errors when we're deleting children of resources, I think it would be more appropriate for the caller of this function to make that decision. For now I have the errors going back in the response because it appears to be consistent with the datastore client.

Reply:
I don't recall why I did this, so I think it's fine to always return errors until we see a problem caused by that.